core/modules/system/system.module

```diff
@@ -4146,12 +4146,12 @@ function path_autocomplete($string = '') {
 
   if ($string) {
     // Get node matches.
-    $results = db_select('node')
-      ->fields('node', array('nid', 'type', 'title', 'status'))
-      ->condition('title', '%' . db_like($string) . '%', 'LIKE')
-      ->condition('status', 1)
+    $results = db_select('node', 'n')
```

Comment:
Instead of creating a brand new alias for node, I propose using 'node', which was the default alias created before this change.

```suggestion
    $results = db_select('node')
```